### PR TITLE
ESD-485: Align chapter capitals and section headings

### PR DESCRIPTION
**Pull request by jehovahmidara**
Signed-off-by: Ogunreku Praise 35612274+jehovahmidara@users.noreply.github.com

### CODE REVIEW

File: Documentation/Guides/UserGuide.md [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
 This user guide describes the general workflow using the HERE Mobile SDK UI Kit and its components in detail. If you are looking for a quick overview, please look at our [README](../../README.md), our [Quick Start](QuickStart.md) guide or the latest _Release Notes_.
 
 ## Contents
-- [Why use HERE Mobile SDK UI Kit?](#why-use-here-mobile-sdk-ui-kit)
+
+- [Why use the HERE Mobile SDK UI Kit?](#why-use-the-here-mobile-sdk-ui-kit)
 - [Where to start?](#where-to-start)
 - [How to read this guide?](#how-to-read-this-guide)
-- [Getting Started - A HERE Mobile SDK UI Kit Primer](#getting-started---a-here-mobile-sdk-ui-kit-primer)
+- [Getting Started - A HERE Mobile SDK UI Kit Primer](#getting-started-a-here-mobile-sdk-ui-kit-primer)
```
Comment (datasun) [inline Documentation/Guides/UserGuide.md]: This link will be broken, because of the missing "-", or?

File: Documentation/Guides/UserGuide.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
 - [Overview of the HERE Mobile SDK UI Kit Primer example](#overview-of-the-here-mobile-sdk-ui-kit-primer-example)
 - [Designing the app flow](#designing-the-app-flow)
 - [Adding HERE Mobile SDK UI Kit components](#adding-here-mobile-sdk-ui-kit-components)
-  - [Adding the map view](#adding-the-map-view)
+	- [Adding the MapView](#adding-the-mapview)
```
Comment (datasun) [inline Documentation/Guides/UserGuide.md]: NMAMapView ... or keep it like it was before ;-)

File: Documentation/Guides/UserGuide.md (hunk 1 above)
Comment (jehovahmidara) [reply Documentation/Guides/UserGuide.md]: No, it is not affected by the new changes I made.

File: Documentation/Guides/UserGuide.md (hunk 2 above)
Comment (jehovahmidara) [reply Documentation/Guides/UserGuide.md]: I resolve to keeping it as it were before

File: Documentation/Guides/UserGuide.md (hunk 1 above)
Comment (jehovahmidara) [reply Documentation/Guides/UserGuide.md]: I will address this in an amended commit.

**Pull request closed without merging**